client/asset/dcr/rpcwallet.go

```diff
@@ -1216,3 +1255,23 @@ func isAccountLockedErr(err error) bool {
 	return errors.As(err, &rpcErr) && rpcErr.Code == dcrjson.ErrRPCWalletUnlockNeeded &&
 		strings.Contains(rpcErr.Message, "account is already locked")
 }
+
+// newWalletInfo is walletinfo with a new field found in version 9.2.0+.
+//
+// TODO: Just use *walletjson.WalletInfoResult after we update to dcrwallet/v4.
+type newWalletInfo struct {
+	*walletjson.WalletInfoResult
+	VSP string `json:"vsp"`
+}
+
+func (w *rpcWallet) walletInfo(ctx context.Context) (*newWalletInfo, error) {
+	var walletInfo newWalletInfo
+	err := w.rpcClientRawRequest(ctx, methodWalletInfo, nil, &walletInfo)
+	return &walletInfo, translateRPCCancelErr(err)
+}
+
+var _ ticketPager = (*rpcWallet)(nil)
+
+func (w *rpcWallet) TicketPage(ctx context.Context, scanStart int32, n, skipN int) ([]*asset.Ticket, error) {
+	return make([]*asset.Ticket, 0), nil
+}
```

Comment:
Without this there will be an error:
```
2024-05-07 08:50:36.970 [ERR] WEB: error retrieving ticket page for 42: ticket pagination not supported for this wallet
```

But it doesn't look like the pager is needed. Should this error be silence somewhere else?

Also the statuses are all unknown, which I think is a separate issue.

Reply:
Without `TicketPage` will it show the tickets that have already voted?

> Also the statuses are all unknown, which I think is a separate issue.

The native wallet doesn't show undefined.

Reply:
I guess we should consider using the tx history db for paginating ticket history. At least for spv wallet but maybe for all. No need to do it in this PR though. Sounds like a bit of work.

Reply:
> Without TicketPage will it show the tickets that have already voted?

The already voted tickets always seem to go away. Without being a `TicketPage` you can't view any pages with under 10 tickets.